### PR TITLE
2.3.4 release Blogpost

### PR DESCRIPTION
**Pull request by Swiftb0y**
Based on #288 so pre-commit works.

### CODE REVIEW

File: content/news/XXXX-XX-XX-mixxx-2-3-4-released.md
Comment (daschuer) [inline content/news/XXXX-XX-XX-mixxx-2-3-4-released.md]:
```suggestion
Let's take a look at the changes:  
```

File: content/news/XXXX-XX-XX-mixxx-2-3-4-released.md
Comment (daschuer) [inline content/news/XXXX-XX-XX-mixxx-2-3-4-released.md]:
```suggestion
macOS received a bunch of love, especially in regards to possible UI performance
```

https://en.wikipedia.org/wiki/MacOS

File: content/news/XXXX-XX-XX-mixxx-2-3-4-released.md
Comment (daschuer) [inline content/news/XXXX-XX-XX-mixxx-2-3-4-released.md]:
```suggestion
problems and translation issues. We also enabled support for macOS 13 (Ventura) on Intel-CPUs, while there is still some work is in progress to support Apple silicon from Mixxx 2.4.0. 
```

File: content/news/XXXX-XX-XX-mixxx-2-3-4-released.md
Comment (daschuer) [inline content/news/XXXX-XX-XX-mixxx-2-3-4-released.md]:
```suggestion
In addition, our Tango and LateNight skins received a bunch of minor tweaks
```

**Review by daschuer — CHANGES_REQUESTED** [review]
Thank you for taking care.

File: content/news/XXXX-XX-XX-mixxx-2-3-4-released.md
Comment (ronso0) [reply content/news/XXXX-XX-XX-mixxx-2-3-4-released.md]: "macOS received quite some love"
too many "a bunch"s in this post ; )

File: content/news/XXXX-XX-XX-mixxx-2-3-4-released.md
Comment (ronso0) [reply content/news/XXXX-XX-XX-mixxx-2-3-4-released.md]: > Thank you for taking care.

Yup, thank you!

File: content/news/XXXX-XX-XX-mixxx-2-3-4-released.md
Comment (Swiftb0y) [reply content/news/XXXX-XX-XX-mixxx-2-3-4-released.md]: Yes, I'll reduce them. Bad habit :+1:

**Comment by Swiftb0y** [comment]
Done, changes are in the fixup commit, I'll squash before merge.

File: content/news/XXXX-XX-XX-mixxx-2-3-4-released.md
Comment (ronso0) [inline content/news/XXXX-XX-XX-mixxx-2-3-4-released.md]:
```suggestion
metadata from Rekordbox databases or music files containing Serato metadata.
```
or was there a reason for capitals?

Comment (Swiftb0y) [reply content/news/XXXX-XX-XX-mixxx-2-3-4-released.md]: Good catch, no there really wasn't, I just didn't spot them while proofreading.

Comment (ronso0) [reply content/news/XXXX-XX-XX-mixxx-2-3-4-released.md]: Also applies to Changelog below, but that could also be an exception because it's a title? don't care much, nit-picking..

Comment (Swiftb0y) [reply content/news/XXXX-XX-XX-mixxx-2-3-4-released.md]: You're right, also fixed "GitHub".

**Review by ronso0 — APPROVED** [review]
Looks good, thank you!

**Comment by Swiftb0y** [comment]
Thank you, ready for merge

File: content/news/XXXX-XX-XX-mixxx-2-3-4-released.md
Comment (JoergAtGithub) [inline content/news/XXXX-XX-XX-mixxx-2-3-4-released.md]:
```suggestion
2.3.4](https://github.com/mixxxdj/mixxx/releases/tag/2.3.4) to be exact.
```

**Review by JoergAtGithub — CHANGES_REQUESTED** [review]
Please fix the link - otherwise it is well written!

File: content/news/XXXX-XX-XX-mixxx-2-3-4-released.md
Comment (Swiftb0y) [reply content/news/XXXX-XX-XX-mixxx-2-3-4-released.md]: Ah, copy-paste mistake, thank you. Fixed.

**Comment by daschuer** [comment]
Thank you. Looks good to me. 
We need to put that on hold until we have a tag.
We also need to modify the download links in https://github.com/mixxxdj/website/blob/website/content/pages/download.md
Do we want to put this here as well to have a canonical merge?

**Review by daschuer — COMMENTED** [review]
.

**Comment by Swiftb0y** [comment]
Done, but we have to remember to update the release announcement url this time... :sweat_smile:

**Comment by Swiftb0y** [comment]
as far as I can tell the build failures stem from the fact that we didn't actually tag the release yet.

**Comment by ronso0** [comment]
> We need to put that on hold until we have a tag.

Yup, draft, just to be safe ;)

**Comment by Swiftb0y** [comment]
Release is tagged, build passes.

**Review by daschuer — APPROVED** [review]
Thank you, for the nice  summary.

**Pull request merged**
